content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md

```diff
@@ -1,197 +1,237 @@
-    {{< /text >}}
+Create a `lightstep-secret.yaml` file to hold your access token by copying the following code and replacing `${LS_ACCESS_TOKEN}` with your access token.
 
-## Visualize trace data
+{{< text yaml >}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: lightstep-access-token
+type: Opaque
+data:
+  access_token: ${LS_ACCESS_TOKEN}
+{{< /text >}}
 
-1.  Load the Lightstep [web UI](https://app.lightstep.com/). You'll see the three Bookinfo services listed in the Service Directory.
+## Apply the configurations to your Kubernetes cluster
 
-    {{< image link="./istio-services.png" caption="Bookfinder services in the Service Directory" >}}
+1. Apply the deployment file.
 
-1.  Navigate to the Explorer view.
+    {{< text plain >}}
+    kubectl apply -f otel-collector-deployment.yaml
+    {{< /text >}}
 
-    {{< image link="./istio-explorer.png" caption="Explorer view" >}}
+    Verify that the OpenTelemetry Collector is running:
 
-1.  Find the query bar at the top. The query bar allows you to interactively filter results by a **Service**, **Operation**, and **Tag** values.
+    {{< text plain >}}
+    kubectl get pods -l app=otel-collector
+    {{< /text >}}
 
-1.  Select `productpage.default` from the **Service** drop-down list.
+2. Apply the ConfigMap to your Kubernetes cluster
 
-1.  Click **Run**. You see something similar to the following:
+    {{< text plain >}}
+    kubectl apply -f otel-collector-configmap.yaml
+    {{< /text >}}
 
-    {{< image link="./istio-tracing-list-lightstep.png" caption="Explorer" >}}
+    Verify that the ConfigMap has been created
 
-1.  Click on the first row in the table of example traces below the latency histogram to see the details
-    corresponding to your refresh of the `/productpage`. The page then looks similar to:
+    {{< text plain >}}
+    kubectl get configmap otel-collector-conf
+    {{< /text >}}
 
-    {{< image link="./istio-tracing-details-lightstep.png" caption="Detailed Trace View" >}}
+3. Apply the Secret to your Kubernetes cluster
 
-The screenshot shows that the trace is comprised of a set of spans. Each span corresponds to a Bookinfo service invoked
-during the execution of a `/productpage` request.
+    {{< text plain >}}
+    kubectl apply -f lightstep-secret.yaml
+    {{< /text >}}
 
-Two spans in the trace represent every RPC. For example, the call from `productpage` to `reviews` starts
-with the span labeled with the `reviews.default.svc.cluster.local:9080/*` operation and the
-`productpage.default: proxy client` service. This service represents the client-side span of the call. The screenshot shows
-that the call took 15.30 ms. The second span is labeled with the `reviews.default.svc.cluster.local:9080/*` operation
-and the `reviews.default: proxy server` service. The second span is a child of the first span and represents the
-server-side span of the call. The screenshot shows that the call took 14.60 ms.
+## View metrics in Cloud Observability
```

Comment:
The folder content/en/docs/tasks/observability/distributed-tracing/lightstep is for tracing. Consider placing the metric guide into a different folder.

Reply:
Moved to the metrics section